### ISSUE TITLE
how to implement zero padding?

### ISSUE THREAD

**Issue by nsknsl** [zsdonghao/tensorlayer#94]
Does tensorlayer has some layers like keras's keras.layers.convolutional.ZeroPadding2D(padding=(1, 1), dim_ordering='default') that can do zero padding?

**Comment by zsdonghao** [comment]
do you mean PadLayer?  http://tensorlayer.readthedocs.io/en/latest/modules/layers.html#padding-layer

set the mode='CONSTANT' , it is zero padding.

**Comment by zsdonghao** [comment]
Automatically closing due to lack of recent activity. Please reopen when additional information becomes available. Thanks!
